### ISSUE TITLE
Differentiate internal vs external nodes in machine view

### ISSUE THREAD

**Issue by trombonehero** [cadets/cadets-ui#23]
It would be nice to more clearly differentiate in the high-level machine view between:

- [ ] nodes that have CADETS traces and those that don't
- [ ] nodes that are "ours" (i.e., in our network) vs those that aren't

**Comment by alsz** [comment]
I've differentiated based on the external field, which I believe corresponds to the first checkbox. See the machine-view branch.

**Comment by trombonehero** [comment]
Hi @alsz,

I've pulled in d730a154e783d8a2d34068d9f252b5051622e61c but still see the following:

![internal-external](https://user-images.githubusercontent.com/85833/27741163-c012c9b6-5d8e-11e7-9310-a97dc7fb31c8.png)

Do the nodes look different for you?

**Comment by trombonehero** [comment]
Sorry, the previous screen shot was from the worksheet rather than the machines view. However, I still see the same thing in the machines view:

![screen shot 2017-06-30 at 12 24 17 pm](https://user-images.githubusercontent.com/85833/27741321-4a771788-5d8f-11e7-8937-b6b8cab08229.png)

**Comment by trombonehero** [comment]
Aha:

```diff
--- a/playground/templates/machine-view.html                            [1/6683]
+++ b/playground/templates/machine-view.html
@@ -35,18 +35,20 @@ load_graph_style([ view ]);

 $.getJSON('machines', function(result) {
   for (let machine of result.nodes) {
+    let classes = machine.type;
+    if (machine.external) {
+      classes += ' machine-external';
+    } else {
+      classes += ' machine-internal';
+    }
+
     let node = {
-      classes: machine.type,
+      classes: classes,
       data: machine,
     };
     node.data.label = node_metadata(machine).label;

     graph_node = view.add(node);
-    if (machine.external) {
-        graph_node.addClass('machine-external')
-    } else {
-        graph_node.addClass('machine-internal')
-    }
   }

   for (let e of result.edges) {
```

![screen shot 2017-06-30 at 12 30 57 pm](https://user-images.githubusercontent.com/85833/27741560-0ade9352-5d90-11e7-8ebc-ac73aa4b0b90.png)

Weird that retroactive class modification worked for you but not me. This way seems to work for me too, however, so I'll commit this.

**Comment by trombonehero** [comment]
Resolved by d730a154e783d8a2d34068d9f252b5051622e61c, 091f550a69a60ff6950c6b57d196299417f0090e and b71aea627491bb08791d4d8aa8deb667bacb0efd.
